monitor/flowgen/symbiotic/config.go

```diff
@@ -0,0 +1,28 @@
+package symbiotic
+
+import (
+	"github.com/omni-network/omni/lib/evmchain"
+	"github.com/omni-network/omni/lib/netconf"
+	"github.com/omni-network/omni/lib/tokens"
+	"github.com/omni-network/omni/solver/targets"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type flowConfig struct {
+	srcChain     uint64
+	dstChain     uint64
+	depositToken tokens.Token
+	expenseToken tokens.Token
+	vaultAddr    common.Address
+}
+
+var config = map[netconf.ID]flowConfig{
+	netconf.Omega: {
+		srcChain:     evmchain.IDBaseSepolia,
+		dstChain:     evmchain.IDHolesky,
+		depositToken: tokens.WSTETH,
+		expenseToken: tokens.WSTETH,
```

Comment:
is this really required? Isn't symbiotic always `WSETH` on all chains and all networks?

Reply:
Good question. Theoretically, Symbiotic supports a lot of tokens (see [existing vaults](https://app.symbiotic.fi/deposit?sort=tvl_desc&filter_option=collateral)), but if we can assume that flowgen will never work with other tokens, we can hardcode wstETH. WDYT?